File: Patterns/Manuscript/manuscript/code/java/DesignPatterns/src/main/java/abstractfactory/AdventureLandFactory.java
package abstractfactory;

public class AdventureLandFactory implements AbstractFoodStallFactory {
    public IceCream createIceCream() {
        System.out.println("Creating tropical-themed ice cream");
        return new TropicalIceCream();
    }

    public Pizza createPizza() {
        System.out.println("Creating exotic-themed pizza");
        return new ExoticPizza();
    }
}

File: Patterns/Manuscript/manuscript/code/python/abstractfactory/AdventureLandFactory.py
from TropicalIceCream import TropicalIceCream
from ExoticPizza import ExoticPizza
from AbstractFoodStallFactory import AbstractFoodStallFactory


class AdventureLandFactory(AbstractFoodStallFactory):
    def createIceCream(self):
        print("Creating tropical-themed ice cream")
        return TropicalIceCream()

    def createPizza(self):
        print("Creating exotic-themed pizza")
        return ExoticPizza()
